### ISSUE TITLE
UI does not respond on tap for about 30 seconds after login or after activating application from background

### ISSUE THREAD

**Issue by flexsurfer** [status-im/status-go#308]
It doesn't depend on syncing, looks like during running node it takes 100% CPU

related issue on react-native https://github.com/status-im/status-react/issues/1144

**Comment by tiabc** [comment]
Why do you think it's status-go issue? Or is it something like investigate together?

**Comment by flexsurfer** [comment]
Yeah, we should investigate it together

**Comment by flexsurfer** [comment]
But we call `startNode` and i can't see any processes on js side, so i think something happens on go side

**Comment by divan** [comment]
Is this related to that fixes issue? https://github.com/status-im/status-go/issues/251
Can you still reproduce it?

**Comment by flexsurfer** [comment]
better to ask @annadanchenko

**Comment by flexsurfer** [comment]
@divan still reproduced in latest develop

**Comment by flexsurfer** [comment]
https://www.youtube.com/watch?v=HiYGUkDGjDo interesting results, same device , same app, but different account

**Comment by flexsurfer** [comment]
https://app.testfairy.com/projects/4803622-status/builds/6709709/sessions/71/?accessToken=QWxSetCs5E-PVNjORLJdGO4Hx/s

**Comment by flexsurfer** [comment]
```00:46 D/ReactNativeJS: DEBUG [status-im.protocol.web3.transport:15] - :post-message {:sig "0x048f7d5d4bda298447bbb5b021a34832509bd1a8dbe4e06f9b7223d00a59b6dc14f6e142b21d3220ceb3155a6d8f40ec115cd96394d3cc7c55055b433a1758dc74", :topic "0xaabb11ee", :payload "0x7b3a6d6573736167652d69642022313439393935383632363437392d61393432656439332d376166342d353663352d626535312d613337636538353430303838222c203a72657175697265732d61636b3f20747275652c203a74797065203a6d6573736167652c203a636f6e74656e74207b3a636f6d6d616e6420226c6f636174696f6e222c203a706172616d73207b2261646472657373222022d0a1d0bed0bbd0b4d0b0d182d181d0bad0b0d18f20d183d0bbd0b8d186d0b02c204d6f736b76612c20d09cd0bed181d0bad0bed0b2d181d0bad0b0d18f20d0bed0b1d0bbd0b0d181d182d18c2c2052757373696126616d703b33372e37303230343726616d703b35352e373635303136222c203a626f742d6462206e696c7d2c203a68616e646c65722d64617461206e696c2c203a747970652022636f6d6d616e64222c203a636f6e74656e742d636f6d6d616e6420226c6f636174696f6e222c203a626f7420226d61696c6d616e227d2c203a636f6e74656e742d747970652022636f6d6d616e64222c203a636c6f636b2d76616c75652034362c203a73686f773f20747275652c203a74696d657374616d7020313439393935383632363831327d0a", :symKeyID nil, :pubKey nil, :ttl 120}
00:46 D/ReactNativeJS: DEBUG [status-im.protocol.web3.delivery:141] - :attempt-was-made 1499958626479-a942ed93-7af4-56c5-be51-a37ce8540088
00:46 D/ReactNativeJS: DEBUG [status-im.protocol.web3.transport:15] - :post-message {:sig "0x048f7d5d4bda298447bbb5b021a34832509bd1a8dbe4e06f9b7223d00a59b6dc14f6e142b21d3220ceb3155a6d8f40ec115cd96394d3cc7c55055b433a1758dc74", :topic "0xaabb11ee", :payload "0x7b3a6d6573736167652d69642022313439373533303335343136372d37613362306535382d356234322d356365312d613335662d353235663633356163393861222c203a72657175697265732d61636b3f20747275652c203a74797065203a6d6573736167652c203a74696d657374616d7020313439373533303335343136382c203a636f6e74656e74202248657920626c6564222c203a636f6e74656e742d747970652022746578742f706c61696e222c203a636c6f636b2d76616c756520312c203a73686f773f2066616c73657d0a", :symKeyID nil, :pubKey nil, :ttl 120}
00:46 D/ReactNativeJS: DEBUG [status-im.protocol.web3.delivery:141] - :attempt-was-made 1497530354167-7a3b0e58-5b42-5ce1-a35f-525f635ac98a
00:46 W/ReactNativeJS: WARN [status-im.protocol.web3.delivery:151] - :shh-post-error Error: invalid argument 0: json: cannot unmarshal non-string into Go value of type hexutil.Bytes {:sig "0x048f7d5d4bda298447bbb5b021a34832509bd1a8dbe4e06f9b7223d00a59b6dc14f6e142b21d3220ceb3155a6d8f40ec115cd96394d3cc7c55055b433a1758dc74", :topic "0xaabb11ee", :payload "0x7b3a6d6573736167652d69642022313439393935383631303636302d32303362323630372d383964372d353838372d393665332d333164643535663433336164222c203a72657175697265732d61636b3f20747275652c203a74797065203a6d6573736167652c203a636f6e74656e74207b3a636f6d6d616e6420226c6f636174696f6e222c203a706172616d73207b2261646472657373222022d0a1d0bed0bbd0b4d0b0d182d181d0bad0b0d18f20d183d0bbd0b8d186d0b02c204d6f736b76612c20d09cd0bed181d0bad0bed0b2d181d0bad0b0d18f20d0bed0b1d0bbd0b0d181d182d18c2c2052757373696126616d703b33372e37303230343726616d703b35352e373635303136222c203a626f742d6462206e696c7d2c203a68616e646c65722d64617461206e696c2c203a747970652022636f6d6d616e64222c203a636f6e74656e742d636f6d6d616e6420226c6f636174696f6e222c203a626f7420226d61696c6d616e227d2c203a636f6e74656e742d747970652022636f6d6d616e64222c203a636c6f636b2d76616c75652034322c203a73686f773f20747275652c203a74696d657374616d7020313439393935383631303935377d0a", :symKeyID nil, :pubKey nil, :ttl 120}
00:46 D/ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :status-im.protocol.handlers/post-error
00:46 I/ReactNativeJS: [Error: invalid argument 0: json: cannot unmarshal non-string into Go value of type hexutil.Bytes]
00:46 W/ReactNativeJS: WARN [status-im.protocol.web3.delivery:151] - :shh-post-error Error: invalid argument 0: json: cannot unmarshal non-string into Go value of type hexutil.Bytes {:sig "0x048f7d5d4bda298447bbb5b021a34832509bd1a8dbe4e06f9b7223d00a59b6dc14f6e142b21d3220ceb3155a6d8f40ec115cd96394d3cc7c55055b433a1758dc74", :topic "0xaabb11ee", :payload "0x7b3a6d6573736167652d69642022313439393037373535313531312d36316165616161392d316436342d353561312d386533312d306334343761376431383365222c203a72657175697265732d61636b3f20747275652c203a74797065203a6d6573736167652c203a636f6e74656e74207b3a636f6d6d616e6420226c6f636174696f6e222c203a706172616d73207b2261646472657373222022d0a1d0bed0bbd0b4d0b0d182d181d0bad0b0d18f20d183d0bbd0b8d186d0b02c204d6f736b76612c20d09cd0bed181d0bad0bed0b2d181d0bad0b0d18f20d0bed0b1d0bbd0b0d181d182d18c2c2052757373696126616d703b33372e37303230343726616d703b35352e373635303136227d2c203a68616e646c65722d64617461206e696c2c203a747970652022636f6d6d616e64222c203a636f6e74656e742d636f6d6d616e6420226c6f636174696f6e222c203a626f7420226d61696c6d616e227d2c203a636f6e74656e742d747970652022636f6d6d616e64222c203a636c6f636b2d76616c75652031322c203a73686f773f20747275652c203a74696d657374616d7020313439393037373535313735387d0a", :symKeyID nil, :pubKey nil, :ttl 120}
00:46 D/ReactNativeJS: DEBUG [status-im.utils.handlers:22] - Handling re-frame event:  :status-im.protocol.handlers/post-error
00:46 I/ReactNativeJS: [Error: invalid argument 0: json: cannot unmarshal non-string into Go value of type hexutil.Bytes]```

**Comment by influx6** [comment]
Is this still something the Go side will investigate?

**Comment by tiabc** [comment]
Absolutely.

**Comment by flexsurfer** [comment]
andrey [Today at 12:00 PM] 
in internal-go
goteam @janherich am i right we have running jails for every chat? so if i have 20 chats, we have 20 running jails?  may this be a reason of high cpu usage? if we simultaneously call all these jails?


2 replies
adamb [4 minutes ago] 
That’s true. Every chat is a separate jail cell.


adamb [4 minutes ago] 
Yeah, if some JS code is executed against all jails then it might be a CPU spike

**Comment by tiabc** [comment]
#421 in some form may probably be started together with this issue.

**Comment by tiabc** [comment]
@rasom:
> we are using eccjs for messages in group chat, i thought that encryption/decription is slow and consumes CPU. At least it was slow. But maybe not the case

**Comment by janherich** [comment]
@flexsurfer we should not have jail cell per every chat anymore (it was the case before we removed wallet bot and did the command-scope change)
But we have separate jail cell for each bot and DApp.
At least that's true if the jail cells are created by calling `status/parse-jail` from RN side, if not and there is some other mechanism how jail cells are created, then I have no clue :)

**Comment by tiabc** [comment]
@janherich well, this is you who define when and why jail cells are created, we just provide API to do that. For example, `Parse` command creates a jail (`CreateAndInitCell` in the next version) with the given name. If you only need jail cells for bots, just create those cells you need.

If you need some additional behavior, just file an issue to status-go

**Comment by janherich** [comment]
@tiabc Thanks, that's exactly what I needed to know. In that case the answer to question whether we have jail cell for each chat create is no, we don't.

**Comment by annadanchenko** [comment]
Closed initial status-im/status-react#1144 as can't reproduce.
